ActionEnum.cs

```diff
@@ -0,0 +1,6 @@
+﻿public enum action
```

Comment:
in enum , we use pascal casing 
Action
 https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions